app/src/main/java/com/framgia/wsm/data/source/RequestRepository.java

```diff
@@ -34,20 +34,18 @@ public RequestRepository(RequestRemoteDataSource remoteDataSource) {
         return mRemoteDataSource.createFormRequestLeave(request);
     }
 
-    public Observable<BaseResponse<List<Request>>> getListRequestOff(@NonNull int userId) {
+    public Observable<BaseResponse<List<RequestOff>>> getListRequestOff(@NonNull int userId) {
         // TODO: Edit later
         // return mRemoteDataSource.getListRequestOff(userId);
-        List<Request> requests = new ArrayList<>();
+        List<RequestOff> requestOffs = new ArrayList<>();
         for (int i = 0; i < 5; i++) {
-            Request request = new Request();
-            request.setCreatedAt("16/06/2017");
-            request.setStatus(0);
-            request.setFromTime("17/06/2017");
-            request.setToTime("18/06/2017");
-            requests.add(request);
+            RequestOff requestOff = new RequestOff();
+            requestOff.setCreatedAt("16/06/2017");
+            requestOffs.add(requestOff);
         }
-        BaseResponse<List<Request>> baseResponse = new BaseResponse<List<Request>>(requests) {
-        };
+        BaseResponse<List<RequestOff>> baseResponse =
+                new BaseResponse<List<RequestOff>>(requestOffs) {
+                };
```

Comment:
ERROR: 'object def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.